docs/source/whatsnew/version5.rst

```diff
@@ -0,0 +1,43 @@
+============
+ 5.x Series
+============
+
+IPython 5.0
+===========
+
+Released June, 2016
+
+IPython 5.0 now uses `promt-toolkit` for the command line interface, thus
+allowing real multi-line editing and syntactic coloration as you type. 
```

Comment:
I will write a longer description of the PTK work, along with a screenshot, before we properly release.